CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+## Before reporting a bug
+* Update to the latest *development version*.
```

Comment:
I would recommend to caution the user to make a backup of his/her current setup before updating to dev. Even though dev has been mostly stable, there have also been some outliers which caused inability to update or broke some libraries.